sway/desktop/xdg_shell.c

```diff
@@ -146,16 +143,12 @@ static void _close(struct sway_view *view) {
 	}
 }
 
-static void destroy(struct sway_view *view) {
+static void _free(struct sway_view *view) {
 	struct sway_xdg_shell_view *xdg_shell_view =
 		xdg_shell_view_from_view(view);
 	if (xdg_shell_view == NULL) {
```

Comment:
That check is technically no longer required, but I guess it doesn't hurt, just wanted to bring it up